autoload/neomake/makers/ft/elixir.vim

```diff
@@ -54,7 +54,7 @@ endfunction
 
 function! neomake#makers#ft#elixir#mix() abort
     return {
-      \ 'exe' : 'mix',
+      \ 'exe' : 'MIX_ENV=test mix',
```

Comment:
I think this won't work on Windows at least.

Environment variables can be set for jobs for newer Vim/Neovim (https://github.com/neomake/neomake/blob/4bc07569e982004c67089c94b4ccd066c57d0d51/autoload/neomake.vim#L48).
Does `mix` have an option for this maybe?